fuzzing/private/binary.bzl

```diff
@@ -36,6 +36,7 @@ Provider for storing information about a fuzz test binary.
         "corpus_dir": "The directory of the corpus files used as input seeds.",
         "dictionary_file": "The dictionary file to use in fuzzing runs.",
         "engine_info": "The `FuzzingEngineInfo` provider of the fuzzing engine used in the fuzz test.",
+        "options_file": "The .options file to use in OSS-Fuzz.",
```

Comment:
I think attaching an options file to the FuzzingBinary makes sense, but I would like to decouple this concept from OSS-Fuzz as much as possible.

In this case, let's not define this file in terms of how it is being used (which is subject to change in the future).

I would suggest the following description: "A file containing fuzzing engine and sanitizer options to use during execution. The file follows the INI format defined in OSS-Fuzz." Ideally, we'd also include a link to the file format documentation - do we have one?

Reply:
Unfortunately there is no such documentation. `.options` files are only mentioned in passing when the docs talk about shared dictionary files. I like your explanation though and am using it now.